packages/checklist/lib/utils/constants.dart

```diff
@@ -0,0 +1,11 @@
+class Constants {
+  static const String localizationApiPath = 'localization/messages/v1/_search';
```

Comment:
This should be taken from main app. This can be moved to singleton of the package.

Reply:
> This should be taken from main app. This can be moved to singleton of the package.

This is not being used in the package, so we removed it from here